src/api/projectsApi.ts

```diff
@@ -79,20 +80,50 @@ export const fetchGLBUrl = async (
   return url;
 };
 
+export const createProject = async (
+  userId: string,
+  projectName: string,
+  corners: Point2D[],
+): Promise<[boolean, string]> => {
+  try {
+    const project = mapSceneToApiProject(
+      {
+        corners: normalizePoints(corners),
+        objectIds: [],
+        objects: {},
+        selectedObjectId: null,
+      },
+      projectName,
+    );
+    const projectsRef = collection(db, 'users', userId, 'projects');
+    const projectRef = doc(projectsRef);
+
+    await setDoc(projectRef, project);
+
+    return [true, projectRef.id];
+  } catch (error) {
+    console.error('Error creating project:', error);
```

Comment:
Why do we want to move error handling to these functions? They should throw errors that should be handled on higher levels, e.g. by showing a snack bar with the error to the user. Console errors can be used for debugging but we should think about the end user at this stage of the project

Reply:
[AS-113](https://ar-chitect.atlassian.net/browse/AS-113)